### PR TITLE
Non-breaking 1.0.2 validation fixes (et-al affixes, 'container' date)

### PR DESCRIPTION
**Pull request by bwiernik**

### CODE REVIEW

**Comment by adam3smith** [comment]
What's up with the Iberoromanistik style? Those aren't et al changes.

**Comment by bwiernik** [comment]
Good to merge @adam3smith ?

File: universitat-basel-iberoromanistik.csl [hunk 1]
```diff
@@ -28,7 +28,7 @@
         <names variable="translator" delimiter="/" prefix="[" suffix="]">
           <label form="short" suffix=" "/>
           <name delimiter=" / " delimiter-precedes-last="always"/>
-          <et-al prefix=" (" suffix=") " font-style="italic"/>
+          <et-al font-style="italic"/>
```
Comment (adam3smith) [inline universitat-basel-iberoromanistik.csl:31]: Do we need to redefine the term here as for the [] case  above?

File: wiley-vch-books.csl [hunk 2]
```diff
@@ -65,7 +65,7 @@
       <text variable="citation-number" suffix="."/>
       <text macro="authors"/>
       <choose>
-        <if match="all" is-uncertain-date="issued accessed original-date container">
+        <if match="all" is-uncertain-date="issued accessed original-date">
```
Comment (adam3smith) [inline wiley-vch-books.csl:68]: (no need to fix this here, but this also looks wrong...)

**Review by adam3smith — COMMENTED** [review]
One question

File: universitat-basel-iberoromanistik.csl (hunk 1 above)
Comment (bwiernik) [reply universitat-basel-iberoromanistik.csl:31]: No, I checked the documentation and they don't actually put et al in parentheses

File: wiley-vch-books.csl (hunk 2 above)
Comment (bwiernik) [reply wiley-vch-books.csl:68]: As in, the rendering logic is incorrect?

Comment (adam3smith) [reply wiley-vch-books.csl:68]: Yeah. What does in print have to do with uncertain date in general and uncertain access date in particular

**Comment by adam3smith** [comment]
Ok to merge then

**Pull request merged**